tests/core/go_library/BUILD.bazel

```diff
@@ -166,3 +166,45 @@ go_library(
     name = "no_srcs_lib",
     importpath = "github.com/bazelbuild/rules_go/tests/core/no_srcs_lib",
 )
+
+# Tests for go_library and go_source support for .syso srcs
+cc_library(
+    name = "imported",
+    srcs = ["imported.c"],
+    linkstatic = True,
```

Comment:
I suspect this attribute will be going away soon as `cc_static_library` matures over time. Not sure if it's worth adding a second test against `cc_static_library`.

Reply:
I'd prefer to have one or the other for the "imported" library just because it's not the main use case we're trying to solve.  I don't want to distract from the way the `.syso` files are supported in "srcs".

Reply:
`linkstatic` on `cc_library` also has a different purpose as it allows local control over whether this particular library is linked statically into a consumer.

Reply:
Could you move the "this is a hack" comment to this target? We could replace it with `cc_static_library` once that's available in our minimum supported version of Bazel.